cpp/order/Steinhardt.cc

```diff
@@ -0,0 +1,495 @@
+// Copyright (c) 2010-2019 The Regents of the University of Michigan
```

Comment:
Can you retab this file? It looks like in a lot of places you end up with 8 spaces instead of 4 for tabs.

Reply:
Also make sure you follow our (admittedly strange) code style with respect to brace indentation, see [here](https://freud.readthedocs.io/en/latest/sourceconventions.html#cpp)

Reply:
Will do